pybossa/task_creator_helper.py

```diff
@@ -40,19 +40,22 @@ def s3_conn_type():
     return current_app.config.get('S3_CONN_TYPE')
 
 
-def get_task_expiration(current_expiration):
+def get_task_expiration(current_expiration, create_time=None):
     """
     Find the appropriate expiration to be added to a task with expiring data.
     If no expiration is set, return the data expiration; otherwise, return
     the smallest between the current expiration and the data expiration.
     current_expiration can be a iso datetime string or a datetime object
```

Comment:
unrelated to this PR. However, can this comments be updated as well as they seems to be outdated and be updated as per current behavior. Thanks!